inst/include/later.h

```diff
@@ -102,7 +112,7 @@ class BackgroundTask {
     BackgroundTask* task = reinterpret_cast<BackgroundTask*>(data);
     // TODO: Error handling
     task->execute();
-    later(&BackgroundTask::result_callback, task, 0);
+    later(background_task_callback, task, 0);
```

Comment:
Is it not OK in general to pass a C++ static method to `later()`, or is it a problem only in this header file?

Reply:
I think it might not be OK in general. I mean, in the sense that it will trigger a warning on clang-UBSAN. I think the calling convention is not actually different between C/C++ (other than C++ doing name mangling, which is not relevant here).